active_directory/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "0.1.0",
```

Comment:
If we bump to `manifest_version`: _1.0.0_ we can remove `max_agent_version` and `min_agent_version`.